<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShieldCoinsDialog</class>
 <widget class="QDialog" name="ShieldCoinsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shield Coins</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frameZAddress">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="toolTip">
             <string>An optional label to associate with the new receiving address.</string>
            </property>
            <property name="text">
             <string>Z-Address:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="reqShieldAddress">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Pick a private address from the address book.</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="AddressBookButton">
            <property name="toolTip">
             <string>Choose new or previously used address</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../bitcoin.qrc">
              <normaloff>:/images/addressbook1</normaloff>:/images/addressbook1</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Alt+A</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="deleteButton">
            <property name="toolTip">
             <string>Remove this entry</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../bitcoin.qrc">
              <normaloff>:/images/clear</normaloff>:/images/clear</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Use this form to shield transparent coinbase funds by sending to a shielded z-address.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayoutOptions">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Fee</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BitcoinAmountField" name="customFee"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Limit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="operationLimit">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string> utxos</string>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayoutButton">
     <item>
      <widget class="QPushButton" name="shieldButton">
       <property name="toolTip">
        <string>Shield coins</string>
       </property>
       <property name="text">
        <string>Shield</string>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/images/shield</normaloff>:/images/shield</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="toolTip">
        <string>Cancel operation</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/images/clear</normaloff>:/images/clear</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BitcoinAmountField</class>
   <extends>QLineEdit</extends>
   <header>bitcoinamountfield.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
